examples/pm25_simpletest.py

```diff
@@ -1,13 +1,15 @@
 """
-Example sketch to connect to PM2.5 sensor with either I2C or UART.
+Example sketch to connect to PM2.5 sensor with I2C.
```

Comment:
It appears you include the UART code commented out. Why did you change the docstring to remove UART?

Reply:
restaured